app/presentation/RoomItem/index.js

```diff
@@ -320,12 +320,17 @@ export default class RoomItem extends React.Component {
 		);
 	}
 
+	capitalize = (s) => {
```

Comment:
Can you move `capitalize` and `formatDate` away from the component?

Reply:
fixed.